webClient/src/app/core/menu-bar/menu-bar.component.ts

```diff
@@ -297,13 +315,16 @@ export class MenuBarComponent implements OnInit, OnDestroy {
           this.openDirectory();
         } else if (event.altKey && event.which === KeyCode.KEY_K) {
           this.openDatasets();
-        } else if (event.altKey && event.which === KeyCode.KEY_S) {
+        } else if (event.altKey && event.which === KeyCode.KEY_P) {
```

Comment:
All the checks for `event.altKey` are redundant because of filtering athttps://github.com/zowe/zlux-editor/blob/795546b36110362ef206ebe128aa420b3a21177f/webClient/src/app/shared/editor-keybinding.service.ts#L39. You can simplify the code removing `event.altKey` in all conditions.